### DISCUSSION TITLE
paste.rs

### DISCUSSION THREAD

**Discussion by samnystrom** [catppuccin/catppuccin#1915] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

[Stylus](https://github.com/openstyles/stylus)

### Has the requested port been themed already?

https://github.com/Stonks3141/paste.rs

**Comment by sgoudham** [comment]
I'll transfer this issue over to ready for review once the preview screenshots have been uploaded!

**Reply by samnystrom** [reply]
Previews are done!

**Reply by sgoudham** [reply]
Cheers! Transferred to #1925
